WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1683,7 +1683,11 @@ extension ReaderStreamViewController : WPTableViewHandlerDelegate {
 
 
     public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        tableView.deselectRow(at: indexPath, animated: false)
+        if !UIAccessibilityIsReduceMotionEnabled() {
+            // This can conflict with the reduce motion fullscreen push animation
+            tableView.deselectRow(at: indexPath, animated: false)
```

Comment:
Hmm, where else does `deselectRow` end up being called? I wonder if we still need to call this somewhere. I was able to produce a bug in which the cell selection persisted, even after dismissing the fullscreen detail view, with reduced motion. Not able to reproduce, however, but it did go away after cell reuse.

It also looks like we can maybe get away with just calling `tableView.deselectRow(at: indexPath, animated: false)` just after `pushFullscreenViewController`, which appears to have the desired effect in both reduced motion enabled/disabled.